locale/de/LC_MESSAGES/pronterface.po

```diff
@@ -5,181 +5,223 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: Pronterface jm1\n"
-"POT-Creation-Date: 2014-03-30 14:35+CEST\n"
-"PO-Revision-Date: 2012-01-23 10:01+0100\n"
+"POT-Creation-Date: 2021-01-21 02:35+0100\n"
+"PO-Revision-Date: 2021-01-21 06:05+0100\n"
 "Last-Translator: Christian Metzen <metzench@ccux-linux.de>\n"
 "Language-Team: DE <LL@li.org>\n"
-"Language: \n"
+"Language: de\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Generated-By: pygettext.py 1.5\n"
+"X-Generator: Poedit 2.4.2\n"
 
 #: printrun/excluder.py:26
 msgid ""
 "Part excluder: draw rectangles where print instructions should be ignored"
 msgstr ""
+"Part excluder: Zeichnen Sie Rechtecke, bei denen Druckanweisungen ignoriert "
+"werden sollten"
 
 #: printrun/excluder.py:27 printrun/excluder.py:29
 msgid "Reset selection"
-msgstr ""
+msgstr "Auswahl zurücksetzen"
 
-#: printrun/gcodeplater.py:43 printrun/gcodeplater.py:44
+#: printrun/gcodeplater.py:77 printrun/gcodeplater.py:78
 msgid "GCODE files (*.gcode;*.GCODE;*.g)"
-msgstr ""
-
-#: printrun/gcodeplater.py:112 printrun/gcodeplater.py:152
-msgid "Warning: no rotation support for now, object won't be correctly rotated"
-msgstr ""
+msgstr "GCODE Dateien (*.gcode;*.GCODE;*.g)"
 
-#: printrun/gcodeplater.py:141 printrun/gcodeplater.py:178
+#: printrun/gcodeplater.py:199 printrun/gcodeplater.py:236
 msgid "Exported merged G-Codes to %s"
 msgstr ""
 
-#: printrun/gcview.py:352
+#: printrun/gcview.py:106
+msgid "Invalid layer for injection"
+msgstr "Ungültiger Abschnitt angegeben"
```

Comment:
This doesn't seem to be the correct translation - Abschnitt is section rather than layer.